products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
Also related this error  An error occurred: bad URI(is not URI?): images/products/softwarecollections/rhel{tthw-rhelver}/manage-repositories.png
/home/awestruct/gems/bundler/gems/awestruct-1f0bba05cda0/lib/awestruct/handlers/interpolation_handler.rb:24:in `instance_eval'

These don't pop-up in the local docker environment.

I can probably just punt for now and comment out the images.   For many of these there is only one image and it's kind of a nice to have.